### PR TITLE
Function CURRENT_DATABASE() Added to docs

### PR DESCRIPTION
**Pull request by santanu82021**

### CODE REVIEW

**Review by cw00dw0rd — APPROVED** [review]
LGTM

File: 3.10/aql/functions-miscellaneous.md [hunk 1]
```diff
@@ -158,6 +158,16 @@ a request context. Otherwise, the return value of this function will be *null*.
 - returns **userName** (string\|null): the current user name, or *null* if
   authentication is disabled
 
+### CURRENT_DATABASE()
```
Comment (Simran-B) [inline 3.10/aql/functions-miscellaneous.md:161]: Should be placed above CURRENT_USER() for alphabetical order

File: 3.10/aql/functions-miscellaneous.md [hunk 2]
```diff
@@ -158,6 +158,16 @@ a request context. Otherwise, the return value of this function will be *null*.
...
+
+`CURRENT_DATABASE() → databaseName`
+
+Returns the name of the current database.
+
+The current database is the database name that was specified while loging in.
```
Comment (Simran-B) [inline 3.10/aql/functions-miscellaneous.md:167]: This should say that the database is specified in the URL path of the request (or defaults to _system database).

E.g. an AQL query execution in the context of database db1 would be done by using the endpoint `http://host/_db/db1/_api/cursor`

**Review by Simran-B — CHANGES_REQUESTED** [review]
Mostly good. Please address my comments in all edited versions.

**Pull request merged**